src/libutil/hash.hh

```diff
@@ -23,7 +23,7 @@ extern std::set<std::string> hashTypes;
 
 extern const std::string base32Chars;
 
-enum Base : int { Base64, Base32, Base16, SRI };
+enum HashFormat : int { Base64, Base32, Base16, SRI };
```

Comment:
If we're going to rename it, we should use the modern `enum struct` rather than legacy `enum`, which is much less error-prone.
```suggestion
enum struct HashFormat : uint8_t {
    Base64,
    Base32,
    Base16,
    SRI,
};
```

Reply:
I was planning to do that in a new PR, but it can also be done here.

BTW, which keyword is preferred for scoped enumeration is this project? Is it `enum class` or `enum struct`?

Reply:
I prefer `enum struct` as you can't write methods or anything on it anyways.

Reply:
Update:

Adding `HashFormat::` alone involves in 119 substitutions in 37 files.

Should we include them in this PR, or should we proceed in another PR?

Reply:
I think it's fine having it in a separate commit as it's a mechanical change.

Reply:
Addressed.